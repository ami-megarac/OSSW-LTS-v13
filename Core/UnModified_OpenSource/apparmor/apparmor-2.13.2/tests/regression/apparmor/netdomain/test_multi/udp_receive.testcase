send_ip=127.0.0.1
send_port=9100
send_profile=udp_send from 127.0.0.1:9100 to 127.0.0.1:9101,
receive_ip=127.0.0.1
receive_port= 9101
receive_profile=udp_receive from 127.0.0.1:9100 to 127.0.0.1:9101,
message=udp_recv pass
proto=udp
