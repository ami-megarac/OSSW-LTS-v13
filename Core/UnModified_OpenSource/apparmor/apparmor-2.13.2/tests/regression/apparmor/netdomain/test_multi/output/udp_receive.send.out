Sending "udp_recv pass"
